SharpPcap/ICaptureDevice.cs

```diff
@@ -127,6 +127,26 @@ public interface ICaptureDevice
         /// </param>
         void Open(DeviceMode mode, int read_timeout, MonitorMode monitor_mode, uint kernel_buffer_size);
 
+        /// <summary>
+        /// Open the device. To start capturing call the 'StartCapture' function
+        /// </summary>
+        /// <param name="mode">
+        /// A <see cref="DeviceMode"/>
+        /// </param>
+        /// <param name="flags">
+        /// A <see cref="OpenFlags"/>
+        /// </param>
+        /// <param name="read_timeout">
+        /// A <see cref="int"/>
+        /// </param>
+        /// <param name="monitor_mode">
+        /// A <see cref="MonitorMode"/>
+        /// </param>
+        /// <param name="kernel_buffer_size">
+        /// A <see cref="uint"/>
+        /// </param>
+        void Open(DeviceMode mode, OpenFlags flags, int read_timeout, MonitorMode monitor_mode, uint kernel_buffer_size);
```

Comment:
`DeviceMode` and `OpenFlags` are mutually exclusive, you can't use them both to open the device.

Reply:
that is a good point, flags does cover the mode selection which is pretty handy in terms of reducing API scope.